### ISSUE TITLE
Document / obtain documentation for relevant part of UDS

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#21]
## User story
1. As a developer
2. I need documentation on UDS protocol
3. So that I can implement the bootloader functionality according to it

## Acceptance criteria
* Create UDS folder in documentation
* Use markdown wherever possible
* High-Level overview of used parts of UDS (e.g. which of the features do we need, which aren't used)
* Links to documentation of UDS
* Sequence diagram for UDS session starting with MCU in application code and flashing tool disconnected

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
